--- xscreensaver-6.01/hacks/glx/Makefile.in	2021-06-09 16:47:13.738269562 +0100
+++ /home/dave/xscreensaver-6.01-build/xscreensaver-6.01/hacks/glx/Makefile.in	2021-09-01 21:36:45.298103487 +0100
@@ -120,6 +120,7 @@
 		  hypertorus.c glmatrix.c cubestorm.c glknots.c blocktube.c \
 		  flipflop.c antspotlight.c polytopes.c gleidescope.c \
 		  mirrorblob.c blinkbox.c noof.c polyhedra.c polyhedra-gl.c \
+		  commander.c \
 		  antinspect.c providence.c pinion.c involute.c boing.c \
 		  texfont.c carousel.c fliptext.c antmaze.c \
 		  tangram.c tangram_shapes.c crackberg.c glhanoi.c cube21.c \
@@ -169,6 +170,7 @@
 		  hypertorus.o glmatrix.o cubestorm.o glknots.o blocktube.o \
 		  flipflop.o antspotlight.o polytopes.o gleidescope.o \
 		  mirrorblob.o blinkbox.o noof.o polyhedra.o polyhedra-gl.o \
+		  commander.o \
 		  antinspect.o providence.o pinion.o involute.o boing.o \
 		  texfont.o texfont-glsl.o carousel.o fliptext.o antmaze.o \
 		  tangram.o tangram_shapes.o crackberg.o glhanoi.o cube21.o \
@@ -200,6 +202,7 @@
 		  glslideshow jigglypuff klein hypertorus glmatrix cubestorm \
 		  glknots blocktube flipflop antspotlight polytopes \
 		  gleidescope mirrorblob blinkbox noof polyhedra \
+		  commander \
 		  antinspect providence pinion boing carousel fliptext \
 		  antmaze tangram crackberg glhanoi cube21 timetunnel \
 		  juggler3d topblock glschool glcells voronoi moebiusgears \
@@ -253,6 +256,7 @@
 		  rotator.h trackball.h gltrackball.h chessmodels.h \
 		  chessgames.h gllist.h flurry.h tunnel_draw.h ants.h \
 		  polyhedra.h normals.h texfont.h tangram_shapes.h \
+		  commander.h \
 		  sproingies.h extrusion.h glschool.h glschool_gl.h \
 		  glschool_alg.h topblock.h involute.h teapot.h sonar.h \
 		  dropshadow.h starwars.h teapot2.h dnapizza.h curlicue.h \
@@ -273,6 +277,7 @@
 		  cubestorm.man glknots.man blocktube.man flipflop.man \
 		  antspotlight.man polytopes.man gleidescope.man \
 		  mirrorblob.man blinkbox.man noof.man polyhedra.man \
+		  commander.man \
 		  antinspect.man providence.man pinion.man boing.man \
 		  carousel.man fliptext.man antmaze.man tangram.man \
 		  crackberg.man glhanoi.man cube21.man timetunnel.man \
@@ -845,6 +850,9 @@
 polyhedra:	polyhedra.o 	$(PH_OBJS)
 	$(CC_HACK) -o $@ $@.o	$(PH_OBJS) $(HACK_LIBS)
 
+commander:	commander.o 	normals.o $(HACK_TRACK_OBJS)
+	$(CC_HACK) -o $@ $@.o	normals.o $(HACK_TRACK_OBJS) $(HACK_LIBS)
+
 antinspect:	antinspect.o	sphere.o $(HACK_TRACK_OBJS)
 	$(CC_HACK) -o $@ $@.o	sphere.o $(HACK_TRACK_OBJS) $(HACK_LIBS)
 
